### ISSUE TITLE
Series-dependent annotations are not reset

### ISSUE THREAD

**Issue by MoBosse** [open-sdg/open-sdg#1669]
Data repository URL: [https://github.com/DnsUpgradeEnvironment/dns-data](https://github.com/DnsUpgradeEnvironment/dns-data)
Data or metadata issue: meta
Input format: csv

**Describe the bug**
I have an indicator with two series [here](https://dnsupgradeenvironment.github.io/dns-indicators/7-1-ab/).
Each of the series has dependent target lines. The series that is displayed at the start has one target line, the other has three target lines. When loading, the first series is correctly displayed with its target line. If you switch to the other series, it will also be displayed correctly with its three target lines, but if you switch back to the first series, the second and third target line of the second series will still be displayed.

**Additional context**
I am pointing to open-sdg 1.8.0-dev

**Comment by brockfanning** [comment]
@MoBosse A quick note after some testing: From what I can see, this is not happening in 2.0.0-dev. I am guessing that means that it also would not happen in 1.8.0-dev with `chartjs_3: true` - though I have not tested that. Did you happen to have chartjs_3 set to true?

**Comment by MoBosse** [comment]
@brockfanning Yes, you are right. I can confirm that this does not happen with `chartjs_3: true`.

**Comment by brockfanning** [comment]
@MoBosse Would you be OK with closing this, in that case?
